source/includes/steps-fle-configure-the-mongodb-client.yaml

```diff
@@ -115,4 +141,19 @@ content: |
              .build();
            
            MongoClient mongoClient = MongoClients.create(clientSettings);
+     .. tab::
+        :tabid: nodejs
+        
+        .. code-block:: javascript
+           
+          const secureClient = new MongoClient(process.env.MONGO_URL, {
```

Comment:
- `process.env.MONGO_URL` -> `connectionString`
- Same indentation nit as the previous code block (10 spaces -> 11 spaces)

Reply:
Fixed